documentation/advancedDoc/advancedDoc.tex

```diff
@@ -340,6 +340,23 @@ \subsection{Generation of a xml summary of the parameters and variables of a com
 Below is given an example of a file generated by this command based on the model of the first example of section \ref{Dynawo_Advanced_Documentation_Dynawo_executables_Preassembled}:
 \lstinputlisting[language=XML,title=Test.desc.xml]{../resources/exampleExecutables/TestDumpNoExternal/Test.desc.xml}
 
+\subsection[Init models in Dynawo]{Init models in \Dynawo}
+
+In Dynawo we use what we call an init model to initialize some parts of the "real" dynamic model.
```

Comment:
Maybe we should add a deeper explanation on what exactly is the purpose of init models (e.g. initializing all internal variables based on observable outputs/parameters)